core/src/cljstyle/format/core.clj

```diff
@@ -271,23 +271,23 @@
 
 (defn- indent-line
   "Apply indentation to the line beginning at this location."
-  [zloc indents]
-  (let [width (indent/indent-amount zloc indents)]
+  [zloc indents list-indent-size]
```

Comment:
I'll just note here that there's some inconsistency in how this argument is ordered throughout the rest of the changes here. Some places have it as a new final argument like this, others take it as a new second argument. Since this is the more basic of the two configuration arguments here, I think you should consistently place `list-indent-size` before `indents` and (later) the indenter rule arguments. That will also let you keep some `partial` forms intact later on.